### PR TITLE
Add TANH Op to Micro

### PR DESCRIPTION
**Pull request by yair-ehrenwald**
Added TANH operator and test to Tensorflow Lite Micro

### CODE REVIEW

**Comment by yair-ehrenwald** [comment]
@googlebot I signed it!

File: tensorflow/lite/micro/kernels/elementwise_test.cc [hunk 1]
```diff
@@ -160,9 +160,9 @@ TF_LITE_MICRO_TEST(Abs) {
   tflite::testing::TestElementwiseFloat(
       tflite::BuiltinOperator_ABS,  // ABS operator
       {2, 2, 2},                    // Input shape
-      {0.01, -0.01, 10, -10},       // Input values
```
Comment (petewarden) [inline tensorflow/lite/micro/kernels/elementwise_test.cc:163]: Thanks for catching these double promotions! For consistency, can you update all the values to the form x.xf (so 10 would become 10.0f for example). Thanks.

**Review by petewarden — DISMISSED** [review]

**Comment by gbaned** [comment]
@yair-ehrenwald Thank you for your contribution. Can you please sign CLA? Thanks!

**Comment by yair-ehrenwald** [comment]
@googlebot I signed it!

**Comment by yair-ehrenwald** [comment]
@googlebot I fixed it.

**Pull request closed without merging**